src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/ViewModels/PowerLauncherViewModel.cs

```diff
@@ -452,5 +477,19 @@ public bool IsUpToDate(PowerLauncherSettings settings)
         {
             return this.settings.Equals(settings);
         }
+
+        public void SearchPlugins()
+        {
+            if (!string.IsNullOrWhiteSpace(SearchText))
+            {
+                _plugins = new ObservableCollection<PowerLauncherPluginViewModel>(settings.Plugins.Where(p => p.Name.StartsWith(SearchText, StringComparison.OrdinalIgnoreCase)).Select(x => new PowerLauncherPluginViewModel(x, isDark)));
```

Comment:
I'm thinking of whether we should also check if plugin name contains search text, e.g. typing `search` would show `windows search` plugin that way..

Reply:
Agree.

But then we should check for `.contains($" {SearchText}")` that the word in the plugin name has to start with the searched text.